src/lib/style/components/_access-approvals.scss

```
@@ -0,0 +1,21 @@
+@use '../abstracts/variables' as SRC;
+@use 'sass:map';
+
+.AccessApprovalsTable {
+
+    .access-approval-header {
+        background-color: map.get(SRC.$colors, 'gray-200');
+        .thead-cell {
```

Why not this and remove the class?
```suggestion
        th {
```